packages/db-base/src/lib/inMemFileDB.ts

```diff
@@ -171,11 +234,11 @@ class InMemoryFileDB {
         return ret;
     }
 
-    initBackupDir() {
+    initBackupDir(): void {
         this.zlib = this.zlib || require('zlib');
         // Interval in minutes => to milliseconds
         this.settings.backup.period =
-            this.settings.backup.period === undefined ? 120 : parseInt(this.settings.backup.period);
+            this.settings.backup.period === undefined ? 120 : parseInt(String(this.settings.backup.period));
```

Comment:
Wouldn't it make more sense to check `typeof this.settings.backup.period` and react accordingly (default for undefined, parse strings, keep numbers)?

Reply:
my thoughts here were to keep functionality as it is. Even if not defined, parseInt also truncates the floating numbers if existing, maybe not relevant, so basically it does exactly what you want, if undefined, default val is applied, if string it is parsed and if number, it is basically kept.